pygmt/src/grdlandmask.py

```diff
@@ -45,6 +49,41 @@ def grdlandmask(**kwargs):
         in.
     {I}
     {R}
+    {A}
+    resolution : str
+        *res*\[\ **+f**\]. Selects the resolution of the data set to use
+        ((**f**)ull, (**h**)igh, (**i**)ntermediate, (**l**)ow, or
+        (**c**)rude). The resolution drops off by ~80% between data sets.
+        [Default is **l**]. Append **+f** to automatically select a lower
+        resolution should the one requested not be available
+        [abort if not found]. Alternatively, choose (**a**)uto to automatically
+        select the best resolution given the chosen region. Note that because
+        the coastlines differ in details a node in a mask file using one
+        resolution is not guaranteed to remain inside [or outside] when a
+        different resolution is selected.
+    bordervalues : str or list
+        [*values*]. Nodes that fall exactly on a polygon boundary should be
```

Comment:
I'm wondering if the docstring is more clear in this way:
```suggestion
    bordervalues : str or list or bool
        Nodes that fall exactly on a polygon boundary should be
```